### ISSUE TITLE
False positive for rule `comma` inside image literals

### ISSUE THREAD

**Issue by dkiszkiel** [realm/SwiftLint#2345]
### New Issue Checklist

- [✔ ] Updated SwiftLint to the latest version
- [✔ ] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Bug Report

Having  `#imageLiteral(resourceName: "foo,bar,baz")` in your project will trigger a warning, and will be modified when using autocorrect

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint
Linting Swift files at paths
Linting 'ViewController.swift' (1/3)
Linting 'AppDelegate.swift' (2/3)
Linting 'LoadingViewController.swift' (3/3)
/Users/dkiszkiel/Desktop/TestApp/TestApp/AppDelegate.swift:14: warning: Line Length Violation: Line should be 120 characters or less: currently 145 characters (line_length)
/Users/dkiszkiel/Desktop/TestApp/TestApp/ViewController.swift:15:33: warning: Comma Spacing Violation: There should be no space before and one after any comma. (comma)
/Users/dkiszkiel/Desktop/TestApp/TestApp/ViewController.swift:15:37: warning: Comma Spacing Violation: There should be no space before and one after any comma. (comma)
Done linting! Found 3 violations, 0 serious in 3 files.
```

### Environment

* SwiftLint version 0.27.0
* Installation method used: homebrew
* running without config
